src/dwg_api.c

```diff
@@ -23610,6 +23610,11 @@ dwg_add_DIMENSION_ALIGNED (Dwg_Object_BLOCK_HEADER *restrict blkhdr,
                            const dwg_point_3d *restrict text_midpt)
 {
   API_ADD_ENTITY (DIMENSION_ALIGNED);
+  if (dwg->header.version <= R_2_22)
+    {
+      LOG_ERROR ("Invalid entity %s <R7", "DIMENSION_ALIGNED")
```

Comment:
please use lowercase and the public revisions, as in common.c: dwg_versions[]
"<r2.22"

Reply:
No problem, but all version comparisons in this file are like this one.